moonworm/cli.py

```diff
@@ -355,12 +375,22 @@ def generate_argument_parser() -> argparse.ArgumentParser:
         "-p",
         "--project",
         required=True,
-        help=f"Path to brownie project directory",
+        help=f"Path to brownie/foundry project directory",
+    )
+    generate_brownie_parser.add_argument(
```

Comment:
It feels a bit odd to call `generate-brownie `with a foundry arg. Can we change the name of the method to something more general? If this is too disruptive, we could just expose a duplicate method and say that `generate-brownie` will eventually be deprecated.